.github/workflows/unit-tests.yml

```diff
@@ -55,6 +55,12 @@ jobs:
       run: |
         coverage run --source="./matchmaker","./seqr","./reference_data" --omit="*/migrations/*","*/apps.py" manage.py test -p '*_tests.py' -v 2 reference_data seqr matchmaker
         coverage report --fail-under=99
+    - name: Run coverage tests for panelapp
+      env:
+        ENABLE_PANEL_APP: "True"
+      run: |
+        coverage run --source="./panelapp" --omit="*/migrations/*","*/apps.py" manage.py test -p '*_tests.py' -v 2 panelapp
+        coverage report --fail-under=99
```

Comment:
Yeah, I initially tried something like `@mock.patch` but it didn't work because the tests required `panelapp` DB migrations.  There are ways to mock in `settings.installed_apps` too but it made the test a bit cumbersome so I thought this was a cleaner approach.